lib/Serialization/ScanningLoaders.cpp

```diff
@@ -174,10 +174,6 @@ SwiftModuleScanner::scanInterfaceFile(Twine moduleInterfacePath,
           Args.push_back("-Xcc");
           Args.push_back("-fno-implicit-module-maps");
         }
-        for (const auto &candidate : compiledCandidates) {
-          Args.push_back("-candidate-module-file");
```

Comment:
Is `candidate_module_file` dead now? Maybe we can remove it from FrontendOptions.td

Reply:
I decide to preserve the flag and have an option to choose something that is very close to the original behavior. If everything goes well, we can remove the flag in the future.